FueledUtils/UIExtensions.swift

```diff
@@ -114,68 +150,190 @@ public extension UIView {
+	///
+	public func deselectAllRows(animated: Bool) {
 		if let indexPaths = self.indexPathsForSelectedRows {
-			for indexPath in indexPaths {
-				self.deselectRow(at: indexPath, animated: animated)
-			}
+			self.deselectRows(indexPaths, animated: animated)
 		}
 	}
 }
 
 public extension UICollectionView {
+	///
+	/// Deselect given items, optionally with an animation.
+	///
+	/// - Parameters:
+	///   - indexPaths: The items to deselect.
+	///   - animated: `true` if you want to animate the deselection, and `false` if the change should be immediate.
+	///
+	public func deselectItems(_ indexPaths: [IndexPath], animated: Bool) {
+		for indexPath in indexPaths {
+			self.deselectItem(at: indexPath, animated: animated)
+		}
+	}
+
+	///
+	/// **Deprecated**: Please use `deselectAllItems(animated:)` instead.
+	///
+	/// Refer to the documentation for `deselectAllItems(animated:)` for more info.
+	///
+	@available(*, deprecated, renamed: "deselectAllItems(animated:)")
 	public func deselectAllItems(_ animated: Bool) {
+		self.deselectAllItems(animated: animated)
+	}
+
+	///
+	/// Deselect all currently selected rows, optionally with an animation.
+	///
+	/// - Parameters:
+	///   - animated: `true` if you want to animate the deselection, and `false` if the change should be immediate.
+	///
+	public func deselectAllItems(animated: Bool) {
 		if let indexPaths = self.indexPathsForSelectedItems {
-			for indexPath in indexPaths {
-				self.deselectItem(at: indexPath, animated: animated)
-			}
+			self.deselectItems(indexPaths, animated: animated)
 		}
 	}
 }
 
 public extension UIImage {
-	public static func draw(size: CGSize, opaque: Bool = false, scale: CGFloat = 0, graphics: (CGContext) -> Void) -> UIImage {
-		var image: UIImage?
+	///
+	/// Create a `CGContext` allowing to do custom drawing, and returns the resulting image as drawn in the context.
+	/// - Parameters:
+	///   - size: Please refer to the parameters documentation for `UIGraphicsBeginImageContextWithOptions` for more info.
+	///   - siopaqueze: Please refer to the parameters documentation for `UIGraphicsBeginImageContextWithOptions` for more info.
+	///   - scale: Please refer to the parameters documentation for `UIGraphicsBeginImageContextWithOptions` for more info.
+	///   - graphics: The drawing actions to execute.
+	///
+	public static func draw(size: CGSize, opaque: Bool = false, scale: CGFloat = 0.0, graphics: (CGContext) -> Void) -> UIImage {
+		var image: UIImage!
 		autoreleasepool {
 			UIGraphicsBeginImageContextWithOptions(size, opaque, scale)
 			graphics(UIGraphicsGetCurrentContext()!)
 			image = UIGraphicsGetImageFromCurrentImageContext()
 			UIGraphicsEndImageContext()
 		}
-		return image!
+		return image
 	}
 
+	///
+	/// **Deprecated**: Please use `withAlpha(_:)` instead.
+	///
+	/// Refer to the documentation for `withAlpha(_:)` for more info.
+	///
+	@available(*, deprecated, renamed: "withAlpha(_:)")
 	public func imageWithAlpha(_ alpha: CGFloat) -> UIImage {
```

Comment:
Made this method and the next few ones more "swifty"

Reply:
Could be `with(alpha: CGFloat)` ? Although, the similar `UIColor` function  is `withAlphaComponent(_ alpha: CGFloat)`. We could meet halfway with `withAlpha(_ alpha: CGFloat)`